18-hlt-intro.md

```diff
@@ -60,54 +69,56 @@ report = reports.decReport('Hlt1TrackAllL0Decision')
 print report.decision()
 ~~~
 
-The HLT1 selection that was most efficient for hadronic charm and beauty decays
-in Run-I was called Hlt1TrackAllL0. Use the advance function to find an event
-that was accepted by that trigger selection.
+The HLT1 selections that are most efficient for hadronic charm and beauty decays
+in Run-II are called Hlt1TrackMVA and Hlt1TwoTrackMVA. Use the advance function
+to find an event that was accepted by either of these trigger selections.
```

Comment:
Used “Run-II” here but “Run II” earlier (no hyphen). I prefer “Run 2” anyway 😉  but should use the same throughout